internal/datacoord/stats_inspector.go

```diff
@@ -197,54 +235,22 @@ func (jm *statsJobManager) triggerTextStatsTask() {
 	}
 }
 
-func (jm *statsJobManager) triggerJsonKeyIndexStatsTask(lastJSONStatsLastTrigger int64, maxJSONStatsTaskCount int) (int64, int) {
```

Comment:
why delete this?

Reply:
@xiaocai2333 will help to add this logic back later.